src/components/ApplicationDetails/tabs/overview/visualization/hooks/useAppWorkflowData.ts

```diff
@@ -120,7 +104,6 @@ export const useAppWorkflowData = (
       ...(componentNodes?.length ? componentNodes : [componentGroup]),
       ...(buildNodes?.length ? buildNodes : [buildGroup]),
       ...applicationIntegrationTestNodes,
-      ...(staticEnvironmentNodes?.length ? staticEnvironmentNodes : [staticEnvironmentGroup]),
       ...(mvpFeature ? [] : releaseNodes?.length ? releaseNodes : [releaseGroup]),
```

Comment:
Should we also remove `mvpFeature` flag?

Reply:
We can remove this in a seperate PR.